docs/integrations/apps/unified-billing/example-queries.mdx

```diff
@@ -64,6 +14,22 @@ The merchant will not be charged during this period. A subscription only generat
 
 Checkout links are valid for 24 hours and expire automatically to prevent misuse.
 
+#### Field Descriptions
+| Field | Description | Trial Behavior
+|:--------|:-----------|:-----------|
+| accountId | The account UUID of the merchant you want to create a checkout for. | |
+| id (product) | The id of your app. | |
+| type (product) | The type of product. In the case for creating an app checkout, this will be `"APPLICATION"` | |
+| productLevel (product) | The level of your product | |
+| id (scope) | Since the type of scope used here will be `STORE`, the scope id will be the store hash of the merchant you are sending the checkout to. | |
+| type (scope) | The type of scope. In this case for creating an app checkout, this will be `"STORE"`. | |
+| interval (pricingPlan) | The interval at which the merchant should be billed. This can be `"ANNUAL"`, `"MONTH"`, `"ONCE"`, `"QUARTER"`, or `"SEMIANNUAL"`. | |
+| value (price) | The amount the merchant will be charged at each interval | |
+| currencyCode (price) | The currency in which the merchant will be charged | |
+| trialDays (pricingPlan) | The number of trial days granted to the merchant | The merchant will not be invoiced until the trial days are over. | |
```

Comment:
I would get rid of the extra column and reword the description to include the note. I think changing the word from `invoiced` to `charged` also makes sense. It's not guaranteed that the merchant will be invoiced the exact day after trial end. They won't be invoiced until their Bill Cycle Day post trial end. However, they will be getting charged. Not sure what works best, I don't feel so strongly about it

`The number of trial days granted. The merchant will not be charged until the trial days are over.`